docs/admin/cluster-troubleshooting.md

```diff
@@ -45,7 +45,76 @@ of the relevant log files.  (note that on systemd based systems, you may need to
 
 ### Worker Nodes
    * /var/log/kubelet.log - Kubelet, responsible for running containers on the node
-   * /var/log/kube-proxy.log - Kube Proxy, responsible for service load balancing
+   * /var/log/kube-proxy.log - Kube Proxy, responsible for service load balancing## A general overview of cluster failure modes
+
+This is an incomplete list of things that could go wrong, and how to deal with them.
+
+Root causes:
+  - VM(s) shutdown
+  - network partition within cluster, or between cluster and users.
+  - crashes in Kubernetes software 
+  - data loss or unavailability of persistent storage (e.g. GCE PD or AWS EBS volume).
+  - operator error misconfigures kubernetes software or application software.
```

Comment:
operator error: done.

removed periods at end of lines (except in a few cases where it seemed necessary) and sort of made the capitalization more consistent